eng/pipelines/global-build.yml

```diff
@@ -92,6 +92,21 @@ jobs:
       buildArgs: -subset mono+libs /p:RuntimeFlavor=Mono
       timeoutInMinutes: 90
 
+#
```

Comment:
BTW, above it says `Build with RuntimeFlavor only. This exercise code paths where only Configuration is specified` but I think that should say `RuntimeFlavor`